docs/user/JavaScriptCompatibility.md

```diff
@@ -219,6 +219,12 @@ The Graal object is available in GraalVM JavaScript by default, unless deactivat
 - If `true`, hot code is compiled by the GraalVM compiler, resulting in high peak performance.
 - If `false`, GraalVM JavaScript will not be optimized by the GraalVM Compiler, typically resulting in lower performance.
 
+### `Graal.registerPromiseRejectionHandler(handler)`
```

Comment:
`registerUnhandledPromiseRejectionHandler` might be a better name, since it only handles unhandled rejections.
An even better name would be `setUnhandledPromiseRejectionHandler` since that makes it more obvious that there can only be one handler (that is replaced if called again) and that it can be used to clear the handler, too (with `undefined`).

Reply:
went with `setUnhandledPromiseRejectionHandler`